lib/code-manager.js

```diff
@@ -143,31 +164,35 @@ export function getBreakpoints() {
   return breakpoints;
 }
 
-function getCurrentCodeCell() {
-  const buffer = store.editor.getBuffer();
-  let start = buffer.getFirstPosition();
+function getCurrentCodeCell(editor: atom$TextEditor) {
+  const buffer = editor.getBuffer();
+  let start = new Point(0, 0);
   let end = buffer.getEndPosition();
-  const regexString = getRegexString(store.editor);
+  const regexString = getRegexString(editor);
 
   if (!regexString) {
-    return [start, end];
+    return new Range(start, end);
   }
 
   const regex = new RegExp(regexString);
-  const cursor = store.editor.getCursorBufferPosition();
+  const cursor = editor.getCursorBufferPosition();
 
-  while (cursor.row < end.row && isComment(cursor)) {
+  while (cursor.row < end.row && isComment(editor, cursor)) {
     cursor.row += 1;
     cursor.column = 0;
   }
 
   if (cursor.row > 0) {
-    buffer.backwardsScanInRange(regex, [start, cursor], ({ range }) => {
+    buffer.backwardsScanInRange(regex, new Range(start, cursor), ({
+      range
+    }) => {
+      // has to be Range
```

Comment:
👍  for using Range here.
Actually `[start, cursor]` is a range compatible array but then we would need to update the flow definitions.

I think the comment can be removed.